ignitionLogic.py

```diff
@@ -0,0 +1,84 @@
+import pandas as pd

```

Comment:
please add timeStamp and batch number while you itrate data in batches. 
batchDetails.